chat-plugins/scavengers.js

```diff
@@ -26,6 +28,38 @@ if (scavengersRoom) {
 	}
 }
 
+let scavengersLB, dataFile = './config/chat-plugins/scavdata.json';
+try {
+	scavengersLB = JSON.parse(fs.readFileSync(dataFile));
+} catch (e) {
+	scavengersLB = {};
+}
+let addPoints = function (username, points) {
```

Comment:
personally, I would make this kind of line ``function addPoints(username, points) {``

I think that's more towards the rest of how PS typically would define this kind of function